mesh_client/__init__.py

```diff
@@ -945,9 +957,14 @@ def __init__(self, msg_id: str, response, client):
             if attribute in _BOOLEAN_HEADERS:
                 header_value = header_value or "N"
                 header_value = header_value.upper() in ["Y", "TRUE"]
+
             setattr(self, attribute, header_value)
         chunk, chunk_count = map(int, headers.get("Mex-Chunk-Range", "1:1").split(":"))
 
+        if not hasattr(self, "content_type") or not self.content_type:  # type: ignore[has-type]
+            # try and get content_type from Mex-Content-Type first, but fallback to request content type if not set
```

Comment:
Is this comment relevant here? We're just setting whatever was decided in `_headers_for_chunk()`